cdist/conf/explorer/disks

```diff
@@ -1,16 +1,26 @@
 #!/bin/sh -e
 
-os=$("$__explorer/os")
+os="$( "$__explorer/os" )"
+
 case "$os" in
+    freebsd)
+        sysctl -n kern.disks
+    ;;
     openbsd)
-        IFS=',' disks=$(sysctl -n hw.disknames)
-        for d in $disks; do
-            echo "${d%%:*}"
-        done | sed -n '/^[sw]d[0-9][0-9]*/p'
+        sysctl -n hw.disknames | grep -Eo '[sw]d[0-9]+' | xargs
     ;;
-
     *)
-        cd /dev || exit 0
-        echo sd? hd? vd?
+        # hopefully everything else is linux
+        if command -v lsblk > /dev/null
+        then
+            # exclude ram disks and cdroms 
+            lsblk -e 1,11 -dno name | xargs
+        else
+            # fallback
+            find /dev \
+                -type b \
+                -regex '/dev/\(hd\|sd\|vd\)[a-z]+' \
```

Comment:
```suggestion
                -regex '/dev/\(fd\|hd\|sd\|vd\)[a-z]+' \
```

Reply:
`-regex` here is inclusive. which one you want? :smile_cat: